scapy/layers/tls/extensions.py

```diff
@@ -175,12 +175,6 @@ def guess_payload_class(self, p):
         return Padding
 
 
-class ServerListField(PacketListField):
-    def i2repr(self, pkt, x):
-        res = [p.servername for p in x]
```

Comment:
Something like `res = [getattr(p, "servername", p) for p in x]` should work too (and it would be more in line with the other extensions). Looking at its base class it seems the idea is to always show names only:

> the final field is showed as a 1-line list rather than as lots of packets